metrics_plan_api.rst

```diff
@@ -0,0 +1,95 @@
+Metrics Plan API
```

Comment:
As mentioned in the SIG call please move the plans into discussions

Reply:
Moved them into #2122